src/config.rs

```diff
@@ -11,55 +11,47 @@ mod default {
   ## Flags are formatted with their name,
   ## followed by some stripes.
   ## Each stripe has a format of `- [ <color hex code>, <height> ]`.
-
   classic:
     - [ "#E50000", 2 ]
     - [ "#FF8D00", 2 ]
     - [ "#FFEE00", 2 ]
     - [ "#028121", 2 ]
     - [ "#004CFF", 2 ]
     - [ "#770088", 2 ]
-
   pastel:
     - [ "#F59679", 2 ]
     - [ "#F9CE7B", 2 ]
     - [ "#FCF37C", 2 ]
     - [ "#9FFD7C", 2 ]
     - [ "#7BA1FB", 2 ]
     - [ "#AC7CFA", 2 ]
-
```

Comment:
The whitespace in the default config is necessary for clarity. Can you add them back?

Reply:
I'm using vscode with prettier, which automatically removed those whitespaces. Yes I agree they're needed for clarity.